megamek/src/megamek/client/ui/swing/ClientGUI.java

```diff
@@ -1641,6 +1641,30 @@ public void deleteAllUnits(Client c) {
         }
         c.sendDeleteEntities(ids);
     }
+    
+    private boolean saveGame() {
```

Comment:
I'm surprised we don't already have a save game dialog.

Reply:
Couldn't find a class for it. The menu bar command also routes to ClientGUI. I just moved it to its own method because I needed it twice.